scripts/windows/bundle.ps1

```diff
@@ -13,17 +13,19 @@ function download_collectd([string]$collectdCommit, [string]$outputDir="$BUILD_D
 }
 
 function get_collectd_plugins ([string]$buildDir=$BUILD_DIR) {
-    mkdir "$buildDir\plugins\collectd" -ErrorAction Ignore
-    $collectdPlugins = Resolve-Path "$buildDir\plugins\collectd"
-    $requirements = Resolve-Path "$scriptDir\..\get-collectd-plugins-requirements.txt"
-    $script = Resolve-Path "$scriptDir\..\get-collectd-plugins.py"
-    $python = "$buildDir\python\python.exe"
-    $env:PYTHONHOME="$buildDir\python"
-    & $python -m pip install -qq -r $requirements
-    & $python $script $collectdPlugins
-    & $python -m pip list
-    # unset the python home enviornment variable
-    Remove-Item Env:\PYTHONHOME
+    try {
```

Comment:
Wrapping the whole thing in `try` and then just rethrowing whatever it catches seems pointless.  Is this to work around some quirk in Powershell?